ophyd/tests/test_ophydobj.py

```diff
@@ -116,6 +116,27 @@ def increment(**kwargs):
     # check multi unsubscribe
     test_obj.unsubscribe(cid)
     test_obj.clear_sub(increment)
+    assert hit == 1
```

Comment:
why this extra line?

Reply:
Just a little extra paranoia, to be sure that unsubscribing doesn’t fire the subscription one last time. (It shouldn’t.)